documentation/sphinx/source/api-python.rst

```diff
@@ -125,6 +125,10 @@ After importing the ``fdb`` module and selecting an API version, you probably wa
 
     .. note:: |network-options-warning|
 
+    .. method :: fdb.options.set_knob("knob_name=value")
```

Comment:
Ah, I see the documentation of the parameter here. I think the Python documentation is written to specify the name of the parameter here, so we would need to document what its legal values are in the blurb below.

Reply:
This will need to be changed to say something like:

```
.. method:: fdb.options.set_knob(knob_value)
```

The Python documentation is written such that expects the parameter value to be a name rather than a string documenting it.